morphic.js

```diff
@@ -7818,10 +7824,40 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
+		this.destroy();
+    }
+}
+            
+
 MenuMorph.prototype.destroy = function () {
+var myself = this,
+    subitems = function(menu) {
+        return menu.children.filter(function (child) {
+                    return (child instanceof MenuItemMorph) &&
+                           (child.doubleClickAction instanceof MenuMorph);
+			} )
```

Comment:
Here? Shouldn't there just be no space? (the one at the end of the return should be there.

I actually thought that putting one here would be a parse error. You can't put `;` in argument lists, right?